ClavaWeaver/resources/clava/test/api/c/results/PassCompositionTest.js.txt

```diff
@@ -1,21 +0,0 @@
-Dummy pass that has received an option object with the value 'bar' for the key 'foo'
```

Comment:
Keep file, this is an important test

Reply:
Re-added in 4612a1f3b and refactored to remove erroneous code in ef6db7680.